testsuite/tests/ui/master/test_tenant.py

```diff
@@ -0,0 +1,107 @@
+"""
+    Testing ground for testing master on tenant creation / edit / deletion ...
+"""
+from testsuite.ui.views.admin.login import LoginView
+from testsuite.ui.views.common.foundation import NotFoundView
+from testsuite.utils import blame
+from testsuite.ui.views.master.audience.tenant import TenantDetailView, TenantEditView
+from testsuite.ui.views.devel import LandingView
+
+
+# pylint: disable=invalid-name
+def check_is_displayed_in_new_tab(browser, new_tab_opener, View, wait=False):
```

Comment:
This is not "check" but "assert". 
Why are you using `View` and not `view -> View`?
Do you really need param `wait` here?

Reply:
`wait` is not needed and the `View` was renamed after the consultation about the naming conventions with @pehala